src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.java

```diff
@@ -0,0 +1,26 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RequireThis"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.requirethis;
+
+// xdoc section -- start
+class Example1 {
+  int a,b,c;
```

Comment:
Please make it:
`int field1, field2, field3`

So it it will be supper clear when argument is overlapping with field.

Reply:
but if we make this change doesn't that remove the violation for "requireThis", since there is a clear distinction and no ambiguity in fields and argument.

Reply:
I meant rename this 3 fields

Reply:
And all related arguments to be fieldX

Reply:
Done.